docs/release_notes/release_notes-0.14.0.rst

```diff
@@ -25,7 +25,7 @@ Key Features and Updates
   * DOCS-#4077: Add release notes template to docs folder (#4078)
   * DOCS-#4082: Add pdf/epub/htmlzip formats for doc builds (#4083)
 * Dependencies
-  *
+  * FIX-#4113, FIX-#4116, FIX-#4115: Fix red CI (#4114)
```

Comment:
just noticed this. Should we include such minor PRs in the release notes? If so, maybe the note should be more elaborated?

Reply:
Let's wait on fully green CI before changing this, we can verify that the actual in-code and deps changes work well, so not to wait for complete CI rerun again after changing this docs thing.

Reply:
@dchigarev CI is green, this line was rephrased.